sql-join/sassy-cast.sql

```diff
@@ -0,0 +1,9 @@
+select "title",
+      "c"."filmId",
+      "c"."actorId",
+      "a"."firstName",
+      "a"."lastName"
```

Comment:
You only need two columns here, not five:

---

![image](https://user-images.githubusercontent.com/16143771/144539527-ac1fa418-c0d1-4064-a081-3d3a610936cf.png)